src/scanner.cc

```diff
@@ -234,7 +235,6 @@ struct Scanner {
           } else {
             stop();
             next();
-
```

Comment:
nit: Doesn't seem like we need this change? Reducing drive-by whitespace changes like this make git history slightly easier to navigate.

Reply:
I fully agree, my bad for being messy on my end and not catching that!

Reply:
No problem! These things feel extra nitpicky and I really should have lint rules to catch them so people can automatically catch them :)